app/reviews/components/ReviewStarInput.tsx

```diff
@@ -17,11 +17,12 @@ export function ReviewStarInput({ orderItem }: ReviewStarInputProps) {
   } = useFormContext()
   const name = `rating-${orderItem.productId}`
   let stars: boolean[] = [true, true, true, true, true]
-  ;(() => {
+  const update = () => {
     for (let i = 0; i < 5; i++) {
       stars[i] = i + 1 <= rating
     }
-  })()
+  }
+  update()
```

Comment:
I mean can't we just execute the for loop directly without a function

Reply:
True